src/setupWorker/glossary.ts

```diff
@@ -11,6 +11,7 @@ export interface SetupWorkerInternalContext {
   worker: ServiceWorker | null
   registration: ServiceWorkerRegistration | null
   requestHandlers: RequestHandler<any, any>[]
+  keepAliveInterval?: ReturnType<typeof setInterval> | number
```

Comment:
Can we use `number` only here? So the `keepAliveInterval` type is compatible with the `clearInterval` argument types?

We can also type it as `number | undefined`, because the interval reference is indeed undefined at some point of time, before the interval gets created in the `start()` call.

Reply:
Yea - the issue is apparently that I needed to use `window.setInterval` - just using `setInterval` is polluted by node types 🤦